README.md

```diff
@@ -17,6 +17,7 @@ The Unity Buy SDK allows Unity developers to query and sell products from Shopif
 - Create a cart
 - Check out via weblink
 - Checkouts via Apple Pay and Safari Web View
+- Checkouts via Chrome Custom Tab
```

Comment:
I think we need to say `Android Chrome Custom Tab` or if it makes more sense `Android Custom Chrome Tabs`.

I wonder if somewhere we should images of what the checkouts look like for these. Because`Custom Chrome Tabs` is a bit ambiguous.